frame/contracts/README.md

```diff
@@ -57,4 +57,19 @@ will make things a lot easier. One such language is [`ink`](https://github.com/p
 which is an [`eDSL`](https://wiki.haskell.org/Embedded_domain_specific_language) that enables
 writing WebAssembly based smart contracts in the Rust programming language.
 
+## Debugging
+
+Contracts can emit messages to the node console when run on a development chain through the
+`seal_println` API. This is exposed in ink! via
+[`ink_env::debug_println()`](https://docs.rs/ink_env/latest/ink_env/fn.debug_println.html).
+
+In order to see these messages the log level for the `runtime::contracts` target needs to be raised
+to at least the `info` level which is the default. However, those messages are easy to overlook
+because of the noise generated by block production. A good starting point for contract debugging
+could be:
+
+```bash
+cargo run --release -- --dev --tmp -lerror,runtime::contracts
```

Comment:
Info logging is enabled by default for substrate. Maybe we should change it to log::debug?

Reply:
Are you sure about this? I ran the node without any logging arguments and the message came out.

Reply:
Ahh. Now I see what you are doing here. Should not have checked this on my phone :P 

But also in general this is more some debug out or not?

Reply:
Okay, if they are reject on productive chains, this is fine